delete some stuff

devoracer opened the pull request.

devoracer committed:
delete some stuff

alexwelch commented on Stravatest.rb:
```diff
@@ -13,9 +13,6 @@
 elev_gainft = (elev_gain * 3.28084).round(2)
 puts "You gained #{elev_gainft}ft of elevation"
-# do this for each of my last 30 activities
-# add comma?
```
Good job removing that comment

devoracer committed:
added loop for max speed

alexwelch commented on Stravatest.rb:
```diff
@@ -24,10 +21,8 @@
-#   max_speedms = activity["max_speed"]
-#   puts "On #{name} your max speed was #{max_speedms}."
-# end
+activities.each do |activity|
```
good job looping

devoracer committed:
puts out max speed for all 30 of history

devoracer committed:
ALL the data

printed out all ride names
figured out how many activities per page and how many pages total
